grafana/dashboards/cmode/security.json

```diff
@@ -1181,7 +1181,7 @@
     },
     {
       "datasource": "${DS_PROMETHEUS}",
-      "description": "This panel requires a cluster with ONTAP 9.10+ and the Harvest REST collector. \n\nThis panel shows volume anti ransomware detail.",
+      "description": "This panel requires a cluster with ONTAP 9.10+ and the Harvest REST collector. \n\nThis panel shows volume anti ransomware detail. \n\nMake sure that autonomous ransomware protection has been enabled.",
```

Comment:
This panel requires the Harvest REST collector and an ONTAP 9.10+ cluster with autonomous ransomware protection enabled.